packages/mdc-layout-grid/_variables.scss

```diff
@@ -36,6 +36,12 @@ $mdc-layout-grid-default-gutter: (
   phone: 16px
 ) !default;
 
+$mdc-layout-grid-column-width: (
```

Comment:
I tried to look up for this pattern but haven't seen any one have this pattern 😞 The thing I search and found that use sass map to define different property based on screen size only appears in `mdc-layout-grid`.

Not sure if I misunderstand what you meant here, could you elaborate a bit?